contrib/_pdnsutil.zsh_completion

```diff
@@ -0,0 +1,305 @@
+#compdef pdnsutil
+
+#
+# put file _pdnsutil somewhere into $fpath,
+#       e.g. /usr/share/zsh/vendor-completions/
+#
+# commands based on Debian Bullseye package pdns-server 4.4
```

Comment:
Can you update this to 4.6? It would be a pity to merge something that is immediately out of date.

Reply:
Thank you, that's a fair point. I have updated the code to reflect pdnsutil 4.6 (and added minor corrections to handle edge cases if no tsig keys or no zones are available yet).

As mentioned, my first completion ever but hope it improves usability of pdnsutil :)